core/templates/dev/head/pages/exploration_editor/editor_tab/InteractionDetailsCacheService.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A state-specific cache for interaction details. It stores customization args
```

Comment:
Just a little change (my bad, I should have provided a better example file, sorry) can you please format this comment as 
`/**
   * @fileoverview Service to provide state-specific cache for ...
   *  ... 
   */`